src/main/java/xbot/common/subsystems/vision/AprilTagVisionIOPhotonVision.java

```diff
@@ -23,11 +23,21 @@
 import java.util.Set;
 import org.photonvision.PhotonCamera;
 
+import dagger.assisted.Assisted;
+import dagger.assisted.AssistedInject;
+
 /**
  * IO implementation for real PhotonVision hardware.
  * Based on the AdvantageKit sample implementation by team 6328.
  */
 public class AprilTagVisionIOPhotonVision implements AprilTagVisionIO {
+
+    public abstract static class Factory {
```

Comment:
Doesn't this need AssistedFactory?

Reply:
I tried that the first time, but it didn't work because it couldn't be provided directly

Reply:
https://github.com/google/dagger/issues/3200